index.html

```diff
@@ -137,35 +148,35 @@ <h2>Projects</h2>
                             <div class="arrow_key"></div>
                         </a>
                     </div>
-
-
                 </div>
             </section>
 
             <section  class="about_section">
-                <div class="project_text text_color text_header_font">
-                    <h2>About</h2>
-                </div>
-                <div class="text_subject text_color text_body_font about_text">
-                    <p>
-                        Hello!I am a software developer! I can 
-                        help you build a product, feature or website.
-                        Take a look of my works. If you like what you see 
-                        and have a project you need coded, don't hesitate 
-                        and contact me.
-                    </p>
+                <div class="order_2">
+                    <div class="project_text text_color text_header_font">
+                        <h2>About</h2>
+                    </div>
+                    <div class="text_subject text_color text_body_font about_text">
+                        <p>
+                            Hello!I am a software developer! I can 
+                            help you build a product, feature or website.
+                            Take a look of my works. If you like what you see 
+                            and have a project you need coded, don't hesitate 
+                            and contact me.
+                        </p>
+                    </div>
+    
+                    <a class="text_button text_body_font about_button">
+                        <p> Get my CV</p>
+                        <div class="download_image"></div>  

```

Comment:
- [ ] Kindly your `about` section overflows the page kindly make it fit the page width and don't overflow.

Your section
![image](https://user-images.githubusercontent.com/35535130/160384452-bb8a9620-966e-4c60-b515-4fd28c80b1c7.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/160384619-e04e7a55-e57d-49b4-912f-af17b60182ad.png)